setup.py

```diff
@@ -8,9 +8,24 @@
 """
 
 import re
+import sys
 
 from setuptools import setup, find_packages
 
+def _get_requirements(path):
```

Comment:
I tried to use same as in other components, so we can fine tune it later with something better, but as you said your idea doesn't work for 26